context/app/static/js/components/detailPage/Attribution/Attribution.tsx

```diff
@@ -1,32 +1,95 @@
 import React from 'react';
 
+import Stack from '@mui/material/Stack';
+
 import { useFlaskDataContext } from 'js/components/Contexts';
 import { DetailPageSection } from 'js/components/detailPage/style';
 import EmailIconLink from 'js/shared-styles/Links/iconLinks/EmailIconLink';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
-
-import { FlexPaper } from './style';
-import SectionItem from '../SectionItem';
+import SummaryPaper from 'js/shared-styles/sections/SectionPaper';
+import LabelledSectionText from 'js/shared-styles/sections/LabelledSectionText';
+import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
+import InfoAlert from 'js/shared-styles/alerts/InfoAlert';
 
 function Attribution() {
   const {
-    entity: { group_name, created_by_user_displayname, created_by_user_email, entity_type },
+    entity: {
+      group_name,
+      created_by_user_displayname,
+      created_by_user_email,
+      entity_type,
+      processing,
+      creation_action,
+      descendants,
+    },
   } = useFlaskDataContext();
 
-  return (
-    <DetailPageSection id="attribution">
-      <SectionHeader iconTooltipText={`Information about the group registering this ${entity_type?.toLowerCase()}.`}>
-        Attribution
-      </SectionHeader>
-      <FlexPaper>
-        <SectionItem label="Group">{group_name}</SectionItem>
-        <SectionItem label="Registered by" ml>
+  const isProcessedDataset = entity_type === 'Dataset' && processing === 'processed';
+  const isVisLiftedDataset = descendants?.find((descendant) => descendant.dataset_type === 'Histology [Image Pyramid]');
+  const isHiveProcessedDataset = isProcessedDataset && creation_action === 'Central Process';
+  const isSupportDataset = entity_type === 'Support';
```

Comment:
These will be nice little utils to keep in mind 👀 Definitely not a must for this PR, but pulling these out into helper functions so we can reuse this logic elsewhere could be useful.

Reply:
Agreed!